app/ide-desktop/lib/dashboard/env.ts

```diff
@@ -0,0 +1,44 @@
+/** @file Read environment variables from a file. */
+import * as fs from 'node:fs/promises'
+import * as path from 'node:path'
+import * as url from 'node:url'
+
+// ===============
+// === loadEnv ===
+// ===============
+
+/** Read environment variables from a file. */
+export async function loadEnv() {
```

Comment:
If it's reading environment from a file then perhaps `readEnvironmentFromFile` would be clearer?